### PR TITLE
01 graphiql route

### PR DESCRIPTION
**Pull request by antonina220590**
#### 🧑‍⚖️ Pull Request Naming Convention

- Title should follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary)
- Do not put issue id in title
- Use [Draft PR](https://github.blog/2019-02-14-introducing-draft-pull-requests/) functionality

* [x] I followed naming convention rules

---

#### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 🔗 Related issue link

_Describe the source of requirement, like related issue link._

#### 💡 Background and solution
- [x] add fetch function 
01_request-response
- [x] add toast for errors display
02_error1
02_error2
- [x] setup redux for headers and variables
- [x] add headers to fetch function. add possibility to add and delete headers 
headers
- [x] add variables to fetch function 
variables

### CODE REVIEW

File: app/graphiQL/page.tsx [hunk 1]
```diff
@@ -68,14 +146,18 @@ export default function GraphiQLClient() {
             <ResizablePanel defaultSize={50}>
               <div className="relative flex h-[100%] items-center justify-center p-6 bg-[#c8c8c8]">
                 <HeadersPanel />
-                <textarea className="w-[100%] h-[100%] bg-[#c8c8c8] text-white font-light"></textarea>
+                <textarea
+                  className="w-[100%] h-[100%] bg-[#c8c8c8] text-white font-light"
```
Comment (KrKate) [inline app/graphiQL/page.tsx]: Maybe you should choose a more contrasting font color (text-white not very visible)?

File: app/graphiQL/page.tsx [hunk 2]
```diff
@@ -68,14 +146,18 @@ export default function GraphiQLClient() {
...
+                  onChange={(e) => setQuery(e.target.value)}
+                ></textarea>
               </div>
             </ResizablePanel>
             <ResizableHandle />
             <ResizablePanel defaultSize={50}>
               <div className="relative flex h-[100%] items-center justify-center p-6 bg-[#c8c8c8] z-20">
                 <SchemaPanel />
-                <span className="font-light">One</span>
+                <span className="font-light">{responseData}</span>
```
Comment (KrKate) [inline app/graphiQL/page.tsx:160]: I suggest adding a fixed height and a scrollbar :)

**Review by KrKate — APPROVED** [review]

File: app/graphiQL/page.tsx (hunk 1 above)
Comment (antonina220590) [reply app/graphiQL/page.tsx]: Done

File: app/graphiQL/page.tsx (hunk 2 above)
Comment (antonina220590) [reply app/graphiQL/page.tsx:160]: It's a good point. I added scrollbar. Thank you for review :)

**Review by InaFK — APPROVED** [review]

**Pull request merged**